package.json

```diff
@@ -113,7 +112,7 @@
     "vm-browserify": "1.1.2"
   },
   "devDependencies": {
-    "@emurgo/cardano-serialization-lib-nodejs": "6.0.0",
+    "@emurgo/cardano-serialization-lib-nodejs": "7.1.0",
```

Comment:
should this be in dependencies instead of devDependencies?

Reply:
nope

Reply:
I think this library is only used for Flow types, that's why it's in devs. It is never used for any actual functionality. This lib is the WASM version of the serialization-lib, which as we discussed we cannot use in the mobiles directly, that's why we have this react-native library in regular dependency, that is the native-wrapped version of that same serialization-lib:

https://github.com/Emurgo/yoroi-mobile/blob/ef47d293a96cb8c4c1139d1fce3b3ed1c3ba28c0/package.json#L37

Reply:
Sorry, now I can answer more in detail. The nodejs libraries are used both in the extension and mobile for jest, not for flow, since jest uses a node runtime environment. 
As a side note, it's important you keep this in mind when adding tests that consume the rust libs: If your test uses functions that are not in `react-native-haskell-shelley`, or that have bugs, you won't notice it because you are actually consuming the nodeJS version (cc @stackchain).